apps/website/foundation/themes/README.md

```diff
@@ -193,7 +174,7 @@ SASS-based theming is deprecated in Clarity 3.0 and will no longer work when Cla
 
 SASS-based theming is most familiar to those who use SASS/SCSS in their own products to build CSS stylesheets. SASS acts like a superset of CSS that enables convenient nesting syntax and programming-like functionality in a CSS-based language that compiles to CSS. SASS also gives developers variables they can reuse throughout their SASS codebase.
 
-Clarity has a number of such variables. The [\_variables.clarity.scss](https://github.com/vmware/clarity/blob/master/src/clr-angular/utils/_variables.clarity.scss) file in the Clarity project can serve as a directory for where all of the variable files can be found.
+Clarity has a number of such variables. The [\_variables.clarity.scss](https://github.com/vmware/clarity/blob/master/packages/angular/projects/clr-angular/src/utils/_variables.clarity.scss) file in the Clarity project can serve as a directory for where all of the variable files can be found.
```

Comment:
Backslash in `[\_variables.clarity.scss]`?

Reply:
The backslash is added by prettier. [playground example](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBtA+gMwJYBs4B0AzmEUQLoAUu2UA1gJQgA0IEADjNtEcqAIYAnQRADuABSEJeKfgDcI2ACYsQAI0H8wdODADK7LbQDmyGIICucVgAsYAW1wB1G9nhFDYOHulvsctwBPZHAyVVoiOEEYcU1je35kTH5cSNYAKyIADwAhTW1dPX57OAAZWjgklLSQTKy9E3wARQsIeCrU6xBDQUjBEITBOiUxKFV2QVoYJ2UYG2QADgAGVgmISKdNdhCJuD65StYAR1b4WI4ZEH4iAFooODglR9VBOBPsV9j+eMSkZM7WJF7NgzJYukRGnAWm1Kn9ql0YPw1DMlHNkAAmVjmfh4EwAYQg9gSISg0EOIAskQAKkiZP8anIrABJKBPWB6MCTTgAQVZehggXwHUiAF8RUA) It doesn’t break the link that gets generated when markdwon is rendered out.

I can live with it. Prettier doesn't let us ignore specific lines, only files so the only other option we have is to start ignoring markdown files.